acceptance/Rakefile

```diff
@@ -274,14 +266,6 @@ EOS
     task :aio => 'ci:check_env' do
       beaker_test(:aio)
     end
-
-    desc <<-EOS
-Run the acceptance tests through Beaker and install from git on the configuration targets.
-#{USAGE}
-EOS
-    task :git => 'ci:check_env' do
-      beaker_test(:git)
```

Comment:
Is there an alternative to this? (To running the tests for a particular revision in a git repo that is.)

Reply:
It never worked, it was brought over from Ruby projects. The alternative is eventually https://github.com/puppetlabs/puppetstein, or more immediately some manual work with puppet-agent.